cipd_packages/codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -416,23 +416,28 @@ update these file paths accordingly.
       'notarytool',
       'info',
       uuid,
-      '--password',
-      appSpecificPassword,
       '--apple-id',
       codesignAppstoreId,
+      '--password',
+      appSpecificPassword,
       '--team-id',
       codesignTeamId,
     ];
 
-    log.info('checking notary status with ${args.join(' ')}');
+    final String argsWithoutCredentials = args
```

Comment:
Since there's a few of these special keys, should we make it an explicit list?

See https://github.com/flutter/cocoon/pull/1008 as an example.

Reply:
Sure, stored passwords in a map, and iterate through the map to replace the passwords. 

personally i think replacing one by one is easier but I am good with either approach. updated tests.